crates/criticalup-cli/Cargo.toml

```diff
@@ -22,23 +22,24 @@ clap = { version = "4.5.21", features = ["std", "derive", "help", "usage"] }
 criticaltrust = { path = "../criticaltrust" }
 criticalup-core = { path = "../criticalup-core" }
 futures.workspace = true
+opener = { version = "0.7.2"}
```

Comment:
Cool, this is also used by `rustup`! :) https://github.com/rust-lang/rustup/blob/cb3556eaf68504e62b4e65a308a14b3a65c0bf1f/Cargo.toml#L52